packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -745,18 +756,6 @@ export class TeamsBotImpl {
     );
     Logger.info(Messages.SuccessfullyProvisionedMsTeamsChannel);
 
-    if (!this.config.scaffold.botId) {
-      this.config.scaffold.botId = botAuthCreds.clientId;
```

Comment:
why remove these？

Reply:
actually, them were moved up to where these configs generated.
The final goal is to save the new configs into conext.